files/pagerduty.rb

```diff
@@ -18,12 +18,18 @@ def api_key
   def trigger_incident
     return false unless api_key
     require 'redphone/pagerduty'
-    Redphone::Pagerduty.trigger_incident(
+    response = Redphone::Pagerduty.trigger_incident(
       :service_key  => api_key,
       :incident_key => incident_key,
-      :description  => description,
+      :description  => description(1024),
       :details      => full_description_hash
-    )['status'] == 'success'
+    )['status']
+    if response != 'success'
```

Comment:
Can you invert the logic so that if the response == success => true, but anything else is false?